test/format/config.c

```diff
@@ -709,6 +709,10 @@ config_in_memory(void)
      * don't have to configure in-memory every time we configure something like LSM, that's too
      * painful.
      */
+    if (config_is_perm("format.abort")) {
+        config_single("runs.in_memory=0", false);
+        return;
+    }
```

Comment:
@etienneptl, I tried the change you tried:
```
diff --git a/test/format/config.c b/test/format/config.c
index 88964902d..33b8d7a91 100644
--- a/test/format/config.c
+++ b/test/format/config.c
@@ -715,6 +715,8 @@ config_in_memory(void)
         return;
     if (config_is_perm("checkpoint"))
         return;
+    if (config_is_perm("format.abort"))
+        return;
     if (config_is_perm("import"))
         return;
     if (config_is_perm("logging"))
```
and it seems to work fine for me, if `format.abort` is set we never configure for an in-memory test.

My test was setting `format.abort=1` on the command line and checking if `in_memory == 0` in the created `RUNDIR/CONFIG` file.

When you tested, you weren't explicitly setting `runs.in_memory` were you?

Reply:
Yes, I was explicitly setting `runs.in_memory=1`, my bad. I will remove `config_single("runs.in_memory=0", false);`

Reply:
1a96c9bd27985c337902552651f4699e6db13942